app/src/main/java/com/mr/nemo/dragonfly/ui/screen/auth/signup/info/SignUpInfoScreen.kt

```diff
@@ -129,16 +161,19 @@ fun SignUpInfoItem(
 fun SignUpInfoScreenPreview() {
     DragonFlyTheme {
         SignUpInfoScreen(
-            items = listOf(
-                InfoItem(
-                    title = "How to reset password",
-                    subtitle = "If you forget your password, you can request a password reset via email or the number you registered"
-                ),
-                InfoItem(
-                    title = "How to create a virtual account",
-                    subtitle = "To create a virtual account you need to raise your customer status to priority"
+            state = SignUpInfoScreenState(
+                items = listOf(
+                    InfoItem(
+                        title = "How to reset password",
+                        subtitle = "If you forget your password, you can request a password reset via email or the number you registered"
+                    ),
+                    InfoItem(
+                        title = "How to create a virtual account",
+                        subtitle = "To create a virtual account you need to raise your customer status to priority"
+                    )
```

Comment:
@bohdanzdaniekin let's extract these titles and subtitiles into resources

Reply:
@slothdeboss There is no sense for that for a while, cause these items used only for composable preview